test/cpp/api/modules.cpp

```diff
@@ -1097,7 +1097,7 @@ TEST_F(ModulesTest, MultiLabelSoftMarginLossWeightedNoReduction) {
   auto input = torch::tensor({{0., 2., 2., 0.}, {2., 1., 0., 1.}}, torch::requires_grad());
   auto target = torch::tensor({{0., 0., 1., 0.}, {1., 0., 1., 1.}}, torch::kFloat);
   auto weight = torch::tensor({0.1, 0.6, 0.4, 0.8}, torch::kFloat);
-  auto options = MultiLabelSoftMarginLossOptions().reduction(Reduction::None).weight(weight);
+  auto options = MultiLabelSoftMarginLossOptions().reduction(at::Reduction::None).weight(weight);
```

Comment:
@yf225  why is it `at::` instead of `toorch::`

Reply:
@divyanshsinghvi thanks a lot for the catch! I changed it to `torch::`